src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -0,0 +1,131 @@
+import { Component, Element, Host, h, Method } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-main-menu',
+  styleUrl: 'biggive-main-menu.scss',
+  shadow: true,
+})
+export class BiggiveMainMenu {
+  @Element() host: HTMLBiggiveHeaderElement;
+
+  openMobileMenu = () => {
+    const mobileMenu = this.host.shadowRoot.querySelector('.nav-links') as HTMLElement;
+    mobileMenu.style.left = '0';
```

Comment:
I think this is nicer / less error-prone if we avoid the `as`:
```suggestion
    this.host.shadowRoot.querySelector<HTMLElement>('.nav-links').style.left = '0';
```

Reply:
Updated in [f8a7bbc](https://github.com/thebiggive/components/pull/99/commits/f8a7bbc1f42f976ff8ec567a1fc7568b47bee0a1) as discussed over Slack